paddle/fluid/operators/distributed_ops/recv_op.cc

```diff
@@ -79,12 +99,21 @@ This operator can get variables from server side.
                  "(int, default 0)"
                  "sync recv or async recv.")
         .SetDefault(0);
+    AddAttr<bool>("with_barrier",
+                  "(bool, default True) if without_barrier=False, will use "
+                  "GetVariable get variable from pserver immediately")
+        .SetDefault(true);
+    AddAttr<std::vector<std::string>>(
+        "varnames",
+        "(string vector, default {})"
+        "sometimes we need to put received var in another name")
```

Comment:
For example, the origin var name is "moment_1.recv@127.0.0.1", the name on the parameter server is "moment_1" actually.
The old "AsyncGetVar" cannot modify var name at the trainer, it will cause some trouble.
The "varnames" is a list of var name.